src/components/toolbar.tsx

```diff
@@ -11,6 +11,7 @@ import { DeleteButton } from "./delete-button";
 import { IToolbarButtonProps, ToolbarButtonComponent } from "./toolbar-button";
 import { EditableTileApiInterfaceRefContext } from "./tiles/tile-api";
 import { kDragTileCreate  } from "./tiles/tile-component";
+import { SectionModelType } from "src/models/curriculum/section";
```

Comment:
should have path starting with `./`

Reply:
Fixed in https://github.com/concord-consortium/collaborative-learning/pull/2458/commits/0fe0e88bc681f100b02fc14112401cf3f0ec54b5